### DISCUSSION TITLE
PCB board

### DISCUSSION THREAD

**Discussion by BIIINO** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#314] [General]
Hello,

is there any option where i can order complete PCB with all necceserry stuff (except ESP i have a lot of them) ?

If yes, please ping me: bino@imafex.sk

Thank you in advance

**Comment by chunkysteveo** [comment]
There's no option here, unless someone is willing to do it for you at a cost, with no guarantee on the board functioning 100%.
